crates/srs/src/voice_stream.rs

```diff
@@ -33,15 +33,18 @@ impl VoiceStream {
     pub async fn new(
         client: Client,
         addr: SocketAddr,
-        recv_voice: bool,
-    ) -> Result<Self, io::Error> {
+        game_source: Option<mpsc::UnboundedReceiver<GameMessage>>
+    ) -> Result<Self, io::Error>
+    {
+        let recv_voice = game_source.is_some();
```

Comment:
Nothing to change here, just a heads-up, I'll probably refactor the Client creation into a builder-pattern, to re-introduce something like `recv_voice` again once I continue working on speech recognition for DATIS.